src/mscorlib/shared/System/Span.NonGeneric.cs

```diff
@@ -22,6 +23,90 @@ namespace System
     /// </summary>
     public static class Span
     {
+        /// <summary>
+        /// Determines whether the beginning of the span matches the specified value when compared using the specified comparison option.
+        /// </summary>
+        public static bool StartsWith(this ReadOnlySpan<char> span, ReadOnlySpan<char> value, StringComparison comparison)
```

Comment:
>l StartsWith(th [](start = 25, length = 15)

All exposed new APIs has to handle the InvariantMode cases and should perform the operations as ordinal/ignorecasing at that time

Reply:
Can you please provide an example of what you mean? We already have StringComparison.InvariantCultureIgnoreCase. What other mode needs to be handled?

Reply:
We added a global flag to .NET Core 2.0 that effectively says to ignore culture, so that even if code uses StringComparison.CurrentCulture, it'll end up using Ordinal, e.g.
https://github.com/dotnet/coreclr/blob/38cf93013c1dd1efc7137a6f4930cab7cc653411/src/mscorlib/shared/System/Globalization/CompareInfo.cs#L339-L347
This is something someone sets when they don't care about cultures and either don't have the needed dependencies (e.g. ICU) or don't want to carry such dependencies due to size constraints.

Reply:
ignore this one. I am seeing we are calling CompareInfo which take care of the InvariantMode. 

To let you know what I mean, you may look https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/Globalization/CompareInfo.cs#L648

Reply:
> We added a global flag to .NET Core 2.0 that effectively says to ignore culture

Good to know. Ty.

> ignore this one.

OK.